docs/bis/guide/other-topics/kindOfQuantities.md

```diff
@@ -0,0 +1,27 @@
+# KindOfQuantities in BIS
+
+The choosing of a [KindOfQuantity](../../ec/kindofquantity.md) in a BIS schema needs to reflect the intended level of coordination of the grouping of properties and default display-formatting that it represents. In general, there are three levels of coordination to keep in mind:
+
+1. Common across the BIS ecosystem.
+1. Common across a particular subset of schemas.
+1. Specific to an iModel Connector or Application.
+
+## Common KindOfQuantities across the BIS ecosystem
+
+KindOfQuantities defined in the [AecUnits](../../domains/AecUnits.ecschema.md) schema shall be referenced by properties whose grouping and default display-formatting are meant to be coordinated across the entire BIS ecosystem, disregarding of the disciplines involved.
+
+As an example, the aecu:LENGTH `KindOfQuantity` can be referenced by properties in schemas at any layer. If the display-formatting of aecu:LENGTH is changed in an Application, the new display-formatting will apply to values from properties across all disciplines referencing it.
+
+## Common KindOfQuantities across a subset of schemas
+
+When the coordination about grouping of properties and default-formatting is only expected among properties in a subset of schemas, it is recommended that a separate schema is introduced to define them.
+
+The [RoadRailUnits](../../domains/RoadRailUnits.ecschema.md) schema is an example of this kind of coordination. For instance, it defines a rru:LENGTH KindOfQuantity that is referenced by properties in Road & Rail schemas whose display-formatting is expected to change together, separate from the common aecu:LENGTH KindOfQuantity mentioned earlier.
+
+## Specific to an iModel Connector or Application
+
+When an iModel Connector or Application faces the need of a special `KindOfQuantity` not covered by existing lower-level schemas, such special `KindOfQuantity` shall be defined in a Connector or Application-specific schema. It is still recommended to introduce these special KindOfQuantities into a separate schema in order to achieve better control over schema upgrades.
```

Comment:
How does this help with schema upgrades?
Do you mean a case when several Application schemas reference same special KoQ schema?

Reply:
I addressed your questions in the following PR: https://github.com/iTwin/itwinjs-core/pull/4893/commits/2bf1c9062c92a0f72f3258b14c77aa9d19caf0cd